### DISCUSSION TITLE
Help needed!!! I can see both stock OSC and uosc in mpv

### DISCUSSION THREAD

**Discussion by bachig26** [tomasklaen/uosc#463] [Q&A] [answered]
I using this commands in `mpv.conf`, so this problem shouldn't arise but I don't know how to solve this.

````
# required so that the 2 UIs don't fight each other
osc=no
# uosc provides its own seeking/volume indicators, so you also don't need this
osd-bar=no
# uosc will draw its own window controls if you disable window border
border=no
````

### Screenshot:

![mpv_JqfzgYhk63](https://user-images.githubusercontent.com/96393348/221967822-272f6711-a8ac-47b2-b667-aa0e1c68375e.png)

**Comment by bachig26** [comment]
[portable_config.zip](https://github.com/tomasklaen/uosc/files/10854417/portable_config.zip)

I'm attaching everything from my mpv config folder, as how it is. 
FYI, I'm using scoop as Installer for mpv that why it's in `~/portable_config` folder

**Comment by tomasklaen** [comment, marked as answer]
Didn't notice anything wrong with your config.

This is double weird since we are also disabling osc internally, so even if you'd not include the `osc=no` in your `mpv.conf`, the `osc` should still get disabled.

Honestly, no idea what's going on. Maybe some weirdness with mpv binaries available in scoop? Have you tried shinchiro's portable build?

**Reply by bachig26** [reply]
if possible, can check if this issue by using with this script https://github.com/christoph-heinrich/mpv-quality-menu.

**Reply by bachig26** [reply]
> Honestly, no idea what's going on. Maybe some weirdness with mpv binaries available in scoop? Have you tried shinchiro's portable build?

I'll try it `shinchiro's portable build` and see how it is.

**Reply by bachig26** [reply]
@tomasklaen I just found the issue and along with comment from https://github.com/christoph-heinrich/mpv-quality-menu author that I included this file `quality-menu-osc.lua` in the scripts folder which caused this overlapping issue.
